beams/beams/custom_scripts/sales_invoice/sales_invoice.py

```diff
@@ -4,12 +4,21 @@
 
 @frappe.whitelist()
 def validate_sales_order_amount(doc, method):
```

Comment:
```suggestion
def validate_sales_order_amount(doc, method):
```
change the method name to reflect what is happening isnude